### PR TITLE
feat: change account deletion message on account settings page

### PR DESCRIPTION
**Pull request by amirtds**

### CODE REVIEW

File: lms/static/js/student_account/components/StudentAccountDeletion.jsx [hunk 1]
```diff
@@ -38,6 +38,13 @@ export class StudentAccountDeletion extends React.Component {
 
   render() {
     const { deletionModalOpen, socialAuthConnected, isActive } = this.state;
+    const appsemblerAccountRemovalText = StringUtils.interpolate(
+      gettext('If you require your account on this site to be deleted, please submit a request via email to {htmlStart}account.deletions@appsembler.com{htmlEnd}. Please submit your request from the email address linked to your user account, and in the email body, include any other profile details you wish to share to enable us to verify your identity, as well as clearly stating that you wish for your account to be deleted. We will endeavor to action your request within 30 days, and intend to further automate this process in the future for a more immediate response. There is no way to undo this request once it has been actioned, and should only be undertaken if absolutely necessary.'),
+      {
+        htmlStart: '<a href="mailto:account.deletions@appsembler.com">',
```
Comment (OmarIthawi) [inline lms/static/js/student_account/components/StudentAccountDeletion.jsx]: How to tell which site the user is coming from?

Comment (bryanlandia) [reply lms/static/js/student_account/components/StudentAccountDeletion.jsx]: Agreed— I think it is actually necessary to have this information, at a minimum which site.  

There's also the possibility that someone's email address in their mail program won't be the one they are using on the Open edX site so if we can get that info that will be helpful.  

```
<a href="mailto:account.deletions@appsembler.com&subject=Account Deletion request {user email address} on  {url-safe Site Name }">
```

**Comment by amirtds** [comment]
@OmarIthawi @bryanlandia  I took care of your comments please let me know if it looks good. thanks

**Comment by OmarIthawi** [comment]
@amirtds there's an error in this PR:


```
WebpackError: 
ModuleBuildError in 
Module build failed: SyntaxError: Unexpected token (44:0)

42 | gettext('If you require your account on this site to be deleted, please submit a request via email to {htmlStart}account.deletions@appsembler.com{htmlEnd}. Please submit your request from the email address linked to your user account, and in the email body, include any other profile details you wish to share to enable us to verify your identity, as well as clearly stating that you wish for your account to be deleted. We will endeavor to action your request within 30 days, and intend to further automate this process in the future for a more immediate response. There is no way to undo this request once it has been actioned, and should only be undertaken if absolutely necessary.'),
43 | {
> 44 | <<<<<<< HEAD
| ^
45 | htmlStart: '<a href="mailto:account.deletions@appsembler.com">',
46 | =======
47 | htmlStart: '<a href="mailto:account.deletions@apps
```

Please double check the sentry logs.

**Pull request merged**